bin/commands/encryptSecrets.js

```diff
@@ -28,8 +28,11 @@ const
   readlineSync = require('readline-sync'),
   ColorOutput = require('./colorOutput');
 
-function commandEncryptSecrets (secretsFile, options) {
-  const cout = new ColorOutput(options);
+function commandEncryptSecrets (file, options) {
+  const
+    vault = new Vault(),
+    secretsFile = file || process.env.KUZZLE_CLEAR_SECRETS_FILE || vault.defaultSecretsFile,
```

Comment:
What is this new environment variable name?
Is this a manual test leftover?

Reply:
I introduce this variable because `KUZZLE_SECRETS_FILE` refers to the encrypted secrets file.  It can be confusing to have the same variable name for both clear and encrypted files.  
I added documentation for this variable in this PR: https://github.com/kuzzleio/documentation/pull/373

Reply:
As discussed, we should remove that environment variable for now.